StreamChatUITestsAppUITests/Tests/MessageList_Tests.swift

```diff
@@ -207,23 +207,23 @@ final class MessageList_Tests: StreamTestCase {
     func test_typingIndicator() {
         linkToScenario(withId: 73)
         
+        let waitTimeout = UIDevice.current.userInterfaceIdiom == .pad ? 10 : XCUIElement.waitTimeout
+        
         GIVEN("user opens the channel") {
-            userRobot
-                .login()
-                .openChannel()
+            userRobot.login().openChannel()
         }
         WHEN("participant starts typing") {
-            participantRobot.wait(2).startTyping()
+            participantRobot.wait(3).startTyping()
         }
         THEN("user observes typing indicator is shown") {
             let typingUserName = UserDetails.userName(for: participantRobot.currentUserId)
-            userRobot.assertTypingIndicatorShown(typingUserName: typingUserName)
+            userRobot.assertTypingIndicatorShown(typingUserName: typingUserName, waitTimeout: waitTimeout)
         }
         WHEN("participant stops typing") {
-            participantRobot.wait(2).stopTyping()
+            participantRobot.wait(3).stopTyping()
```

Comment:
Should we create a variable for these waits? Seems a bit repetitive

Reply:
Done 👍